apps/els_lsp/src/els_rename_provider.erl

```diff
@@ -205,7 +206,8 @@ editable_range(#{kind := Kind, data := #{name_range := Range}}, function) when
     Kind =:= export_type_entry;
     Kind =:= import_entry;
     Kind =:= type_application;
-    Kind =:= type_definition
+    Kind =:= type_definition,
```

Comment:
```suggestion
    Kind =:= type_definition;
```
took me a while...

Reply:
Oh wow haha! that was a subtle bug. Good thing we got tests :)